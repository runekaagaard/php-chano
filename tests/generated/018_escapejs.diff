001- \u003Cscript\u003Eand this\u003C/script\u003E
001+ Fatal error: Call to undefined method DtlIter::escapejs() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/018_escapejs.php on line 4