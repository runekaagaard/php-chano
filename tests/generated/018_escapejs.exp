\u003Cscript\u003Eand this\u003C/script\u003E