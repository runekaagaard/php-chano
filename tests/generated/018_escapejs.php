<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => '<script>and this</script>')));
foreach ($items as $i) echo $i->input->escapejs();
