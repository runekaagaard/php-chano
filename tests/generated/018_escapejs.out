Fatal error: Call to undefined method DtlIter::escapejs() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/018_escapejs.php on line 4